internal/ir/xds.go

```diff
@@ -1062,3 +1071,10 @@ type ConsistentHash struct {
 	// Hash based on the Source IP Address
 	SourceIP *bool `json:"sourceIP,omitempty" yaml:"sourceIP,omitempty"`
 }
+
+// SlowStart defines the slow start configuration.
+// +k8s:deepcopy-gen=true
+type SlowStart struct {
+	// Window defines the duration of the warm up period for newly added host.
+	Window time.Duration `json:"window,omitempty" yaml:"window,omitempty"`
```

Comment:
lets use `*metav1.Duration` here 
also `omitempty` is not needed since its a required field